native-pkcs11-core/src/object.rs

```diff
@@ -74,6 +75,33 @@ impl Clone for Object {
     }
 }
 
+fn extract_ec_params(der_bytes: &[u8]) -> Option<(Vec<u8>, Vec<u8>)> {
+    let spki: SubjectPublicKeyInfoRef<'_> = SubjectPublicKeyInfoRef::try_from(der_bytes).unwrap();
+    // For EC keys, the algorithm parameters contain the curve OID
+    // For EC keys, the subject public key is the EC point
+    Some((
+        ObjectIdentifier::from_bytes(spki.algorithm.parameters.unwrap().value())
```

Comment:
`spki.algorithm.parameters_oid().unwrap().to_der().unwrap()` seems to do the same thing.